### PR TITLE
Pull request cdp

### PR DESCRIPTION
**Pull request by zshaheen**

### CODE REVIEW

**Comment by durack1** [comment]
@zshaheen your commit history tells quite the colourful story..

**Comment by gleckler1** [comment]
@zshaheen  Could we just merge the parser for now?

File: src/python/io/pmp_io.py [hunk 1]
```diff
@@ -395,7 +489,9 @@ def addJson(self, filename):
                     varnm += "-%i" % int(var["level"] / 100.)
             tmp_dict = {varnm: tmp_dict["RESULTS"]}
         else:
-            tmp_dict = tmp_dict["RESULTS"]
+            tmp_dict = tmp_dict["RESULTS
```
Comment (durack1) [inline src/python/io/pmp_io.py]: @zshaheen you're missing a ']' here

Comment (zshaheen) [inline src/python/io/pmp_io.py]: Yeah, I noticed that. Thanks for the heads up.

**Comment by zshaheen** [comment]
@doutriaux1 I just called the base module pmp_io, since all of the other new pmp/cdp/acme models are prefixed as such. Doesn't really matter, so I'll revert it to base.

Also the new driver works with the demo, but I was hesitant to just push the new stuff on the users and completely remove any reference to the old one. If you're cool with it, then it's fine.

I'll fix the stuff you talked about this week and will let you know.

**Comment by doutriaux1** [comment]
@zshaheen yes let's do a complete switch off.

**Comment by zshaheen** [comment]
@doutriaux1 Think this is everything you wanted. Let me know if you want more changes.

**Pull request merged**